keda/values.yaml

```diff
@@ -32,6 +32,9 @@ image:
   # -- Image pullPolicy for all KEDA components
   pullPolicy: Always
 
+# -- Kubernetes cluster name
```

Comment:
Small suggestion on why people would want to specify this:
```suggestion
# -- Kubernetes cluster name. Used in features such as emitting CloudEvents
```

I don't think we should move it under events or so as it might serve a broader purpose in the future